components/SubHeader/index.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="subheader">
+    <h1 class="subheader__title">
+      {{ title }}
+    </h1>
+    <p v-if="descriptions" class="subheader__descriptions">
+      {{ descriptions }}
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ComponentSubHeader',
+  props: {
+    title: {
+      type: String,
+      default: '',
```

Comment:
I think you can omit the default value since this props is required

Reply:
Okay thanks i have done remove required